app/models/user_access.rb

```diff
@@ -72,7 +72,10 @@ class AuthorizationNotPerformedError < StandardError; end
     User.admins.where(organization: user.organization)
   end
 
-  memoize def accessible_users
+  memoize def accessible_users(action)
+    return User.non_admins if bypass?
+    return User.none if action_to_level(action).include?(:manage)
```

Comment:
Do we need to update test coverage for this change? Are we missing test coverage here?

Reply:
We are definitely missing coverage for multiple methods in UserAccess, we're adding that separately today.